amundsen_application/static/js/components/common/Bookmark/MyBookmarks/index.tsx

```diff
@@ -24,29 +29,56 @@ export class MyBookmarks extends React.Component<MyBookmarksProps> {
     super(props);
   }
 
+  generateTabContent = (resource: ResourceType) => {
+    const bookmarks = this.props.myBookmarks[resource] || [];
+    return (
+      <ResourceList
+        allItems={ bookmarks }
+        source={ MY_BOOKMARKS_SOURCE_NAME }
+        itemsPerPage={ BOOKMARKS_PER_PAGE }
+        customEmptyText={ EMPTY_BOOKMARK_MESSAGE }
+      />
```

Comment:
It's unclear from mocks if this section should paginate or click to expand. I tagged you in a comment in Figma.

Reply:
Noted. I will update this in the followup PR base don the clarification;